p2p/connmgr.go

```diff
@@ -48,6 +50,20 @@ func (_ ConnManager) Protect(peer.ID, string)                  {}
 func (_ ConnManager) Unprotect(peer.ID, string) bool           { return false }
 func (_ ConnManager) Close() error                             { return nil }
 
+func Connected(n net.Network, c net.Conn) {
+	// TODO: replace dummy status message with current state
+	status := &StatusMessage{
+		ProtocolVersion:     0,
+		MinSupportedVersion: 0,
+		Roles:               0,
+		BestBlockNumber:     0,
+		BestBlockHash:       common.Hash{0x00},
+		GenesisHash:         common.Hash{0x00},
+		ChainStatus:         []byte{0},
+	}
+	log.Info("connected", "status", status)
```

Comment:
maybe we would want this to be debug level instead of info?

Reply:
I can update / we can discuss in my next pull request.